code/modules/hydroponics/grown.dm

```diff
@@ -60,6 +60,31 @@
 
 /obj/item/reagent_containers/food/snacks/grown/attackby(obj/item/O, mob/user, params)
 	..()
+	if(slices_num > 0 && slice_path)
```

Comment:
Please change this back to `if(slices_num && slice_path)`. As I said on the comment thread False left... he's wrong.